clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java

```diff
@@ -19,39 +19,51 @@
 
 package org.apache.gravitino.cli.commands;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import org.apache.gravitino.cli.ErrorMessages;
 import org.apache.gravitino.client.GravitinoClient;
 import org.apache.gravitino.exceptions.NoSuchMetalakeException;
 import org.apache.gravitino.exceptions.TagAlreadyExistsException;
 
 public class CreateTag extends Command {
   protected final String metalake;
-  protected final String tag;
+  protected final String[] tags;
   protected final String comment;
 
   /**
-   * Create a new tag.
+   * Create tags.
    *
    * @param url The URL of the Gravitino server.
    * @param ignoreVersions If true don't check the client/server versions match.
    * @param metalake The name of the metalake.
-   * @param tag The name of the tag.
+   * @param tags The names of the tags.
    * @param comment The comment of the tag.
    */
   public CreateTag(
-      String url, boolean ignoreVersions, String metalake, String tag, String comment) {
+      String url, boolean ignoreVersions, String metalake, String[] tags, String comment) {
     super(url, ignoreVersions);
     this.metalake = metalake;
-    this.tag = tag;
+    this.tags = tags;
```

Comment:
There is no validation rule for tags, right?
Basically, any character is acceptable?